### PR TITLE
feat(eventstore): add spooler

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: internal/eventstore/spooler/spooler.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+func (s *spooledHandler) query(ctx context.Context) ([]*models.Event, error) {
+	query, err := s.EventQuery()
+	if err != nil {
+		return nil, err
+	}
+	events, err := s.eventstore.FilterEvents(ctx, query)
+	if err != nil {
+		return nil, err
+	}
+	return events, nil
```
Comment (adlerhurst) [inline internal/eventstore/spooler/spooler.go:113]:
```suggestion
	return s.eventstore.FilterEvents(ctx, query)
```

File: internal/eventstore/spooler/spooler_test.go [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+	return h.processError
+}
+func (h *testHandler) MinimumCycleDuration() time.Duration { return h.cycleDuration }
+
+type eventstoreStub struct {
+	events []*models.Event
+	err    error
+}
+
+// Healthz returns status OK as soon as the service started
```
Comment (adlerhurst) [inline internal/eventstore/spooler/spooler_test.go:40]:
```suggestion
// Health returns status OK as soon as the service started
```

File: internal/eventstore/spooler/spooler_test.go [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+func (es *eventstoreStub) Health(ctx context.Context) error {
+	return nil
+}
+
+// Ready returns status OK as soon as all dependent services are available
```
Comment (adlerhurst) [inline internal/eventstore/spooler/spooler_test.go:45]: wrong comment

Comment (fgerschwiler) [reply internal/eventstore/spooler/spooler_test.go:45]: i remove the comments, because its just for tests and i dont really do something in this funcs

**Pull request merged**